crates/sui-adapter-transactional-tests/tests/random/happy_flows.exp

```diff
@@ -10,25 +10,31 @@ gas summary: computation_cost: 1000000, storage_cost: 6209200,  storage_rebate:
 
 task 2 'programmable'. lines 26-29:
 mutated: object(0,0)
+unchanged_shared: 0x0000000000000000000000000000000000000000000000000000000000000008
```

Comment:
Unrelated to this PR really, but I wish we trimmed the leading zeros for these known addresses in the test output